core-bundle/src/Asset/ContaoContext.php

```diff
@@ -23,14 +22,12 @@
 class ContaoContext implements ContextInterface
 {
     private RequestStack $requestStack;
-    private ContaoFramework $framework;
     private string $field;
     private bool $debug;
 
-    public function __construct(RequestStack $requestStack, ContaoFramework $framework, string $field, bool $debug = false)
```

Comment:
IMHO this is a BC break, the constructor is not marked as `@internal` and the class not excluded from our BC promise.

Reply:
it is `@internal`?
https://github.com/contao/contao/blob/14bae73113064e76592035231799b562f29381b6/core-bundle/src/Asset/ContaoContext.php#L20

Reply:
Ah sorry, you are right. I missed the class doc block for some reason.